cluster/gce/configure-vm.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -x
```

Comment:
/cc @zmerlynn 

any reason to this as a set rather than on the top line, e.g. `#!/bin/bash -x`